IBAnimatable/PresentExplodeSegue.swift

```diff
@@ -10,4 +11,8 @@ public class PresentExplodeSegue: UIStoryboardSegue {
     destinationViewController.transitioningDelegate = PresenterManager.sharedManager().retrievePresenter(.Explode(params: []))
     sourceViewController.presentViewController(destinationViewController, animated: true, completion: nil)
   }
+  public func perform(completion: OptionalAnimatableCompletion) {
```

Comment:
I am not sure will we use Segue programmatically like this? Because Apple has default way to use Transition Controllers / Animators which doesn't require a Segue object. `IBAnimtable` provides Segues because we want to provide a convenience way for the user to configure present transition in Interface Builder. I don't think we should use the segue class  programmatically.